serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/model/Entity.java

```diff
@@ -1,28 +1,29 @@
 package no.difi.meldingsutveksling.serviceregistry.model;
 
 
+import com.google.common.base.MoreObjects;
 import no.difi.meldingsutveksling.serviceregistry.servicerecord.ServiceRecord;
 
-import java.util.ArrayList;
-import java.util.List;
-
 /**
  * Represents an organization or a citizen
  */
 public class Entity {
 
     private EntityInfo info;
-    private List<ServiceRecord> serviceRecords;
+    private ServiceRecord serviceRecord;
 
     /**
      * Creates an empty Entity
      */
     public Entity() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1186)